django/db/models/fields/__init__.py

```diff
@@ -2093,6 +2094,10 @@ def get_internal_type(self):
 class TextField(Field):
     description = _("Text")
 
+    def __init__(self, *args, **kwargs):
+        super(TextField, self).__init__(*args, **kwargs)
```

Comment:
Here too. You can use `super().__init__(*args, **kwargs)` as master only supports Python3.